### ISSUE TITLE
problème de question voiture

### ISSUE THREAD

**Issue by nosgestesclimat** [datagir/nosgestesclimat#1836]
On me demande si j'ai une voiture, je réponds "non", et on me demande ensuite sa taille !
> Depuis la page: `https://nosgestesclimat.fr/simulateur/bilan`
> Ce ticket a été créé automatiquement par notre robot depuis notre [page de contribution](https://nosgestesclimat.fr/contribuer).

**Comment by Clemog** [comment]
Oui, si vous faites des kilomètres en voiture, nous avons besoin des caractéristiques de la voiture que vous prenez majoritairement, une notification vous le signale.

On pourrait cependant imaginer un set par défaut pour les personnes qui n'ont pas de voiture et qui ne souhaite pas répondre à toutes ces questions ou alors une granularité intermédiaire pour proposer au moins des actions sur le type de voiture utilisée
